### ISSUE TITLE
Microsoft Azure Version Number Parsing

### ISSUE THREAD

**Issue by catt231** [terraform-providers/terraform-provider-postgresql#50]
### Terraform Version
0.11.8

### Affected Resource(s)
Provider

### Terraform Configuration Files
```hcl
provider "postgresql" {
  alias    = "pg1"
  host     = "server.postgres.database.azure.com"
  username = "user@server"
  password = "***"
}
```

### Debug Output
https://gist.githubusercontent.com/catt231/6719b7b584231bc6702b0ce98a273303/raw/4581446b865fea4079a4009b49aff83dd8730289/postgres-parsing-version

### Expected Behavior
Parse Version Number Successfully

### Actual Behavior
```Error initializing PostgreSQL client: error detecting capabilities: error parsing version: Invalid character(s) found in patch number...```

### Steps to Reproduce
```terraform plan```

### Important Factoids
Azure Postgres `select version()` returns 
"PostgreSQL 9.5.14, compiled by Visual C++ build 1800, 64-bit"

### References
https://serverfault.com/questions/894993/terraform-postgres-provider-not-recognize-azure-postgresql-database-version

**Comment by ordenador** [comment]
i have the same problem

**Comment by carlpett** [comment]
This seems to have been fixed in #40, but there still hasn't been a release with it. @sean-, any plans to make a 0.1.3 soon?

**Comment by carlpett** [comment]
As a workaround for cases where it works, v0.1.0 of this provider *doesn't* attempt to parse the version number, so that is what we've fallen back to for now (but will probably also expose you to some bugs that have since been fixed)

**Comment by ordenador** [comment]
> As a workaround for cases where it works, v0.1.0 of this provider _doesn't_ attempt to parse the version number, so that is what we've fallen back to for now (but will probably also expose you to some bugs that have since been fixed)

in version v.0.1.0:
works when i create the resource, but when i try destroy, appear an error: 
` module.db.postgresql_role.user: postgresql_role.user: dial tcp :5432: getsockopt: connection refused
`

**Comment by sean-** [comment]
Closing as fixed in #40
